### DISCUSSION TITLE
How to gain access to v3 alpha packages on feedz.io

### DISCUSSION THREAD

**Discussion by x-Dave-x** [xunit/xunit#2926] [Question]
According to the doc page for v3

https://xunit.net/docs/v3-alpha

we can gain access to the v3 alpha packages by visiting 

https://feedz.io/org/xunit/repository/xunit/search

However, even after I created an account on that site, all attempts to access 

https://feedz.io/org/xunit/repository/xunit/search

or 

https://feedz.io/org/xunit/repository

fail, the latter with a message that the repository doesn't exist or I don't have access to it. 

Can anyone help me gain access to the v3 alpha packages?

Thanks.

**Comment by bradwilson** [comment]
I can verify that https://feedz.io/org/xunit/repository/xunit/search is available to me:

![image](https://github.com/xunit/xunit/assets/16944/f889a635-36cd-46ef-99aa-a0c64190a4f2)

and the repository is marked as public:

![image](https://github.com/xunit/xunit/assets/16944/e6ffc19b-7b25-4059-ab6a-5531c38e4ed1)

I'm not sure why the web UI isn't working for you.

That said, an account should not be necessary for accessing the feedz.io feed for package restore. You can see instructions here: https://xunit.net/docs/using-ci-builds
